fabischapeli/fast-food-fast-v1
# #160231895 places new meal in the application

What does this PR do?

It adds new meal to the app

Description of Task to be completed?

Write code to write to meal list(adding new meal)

How should this be manually tested?

Fork this branch to your github and clone it to your machine
Go to google chrome, more tools, extensions, add postman 
On postman, make sure your server is running, and include the method and routes you want to test.
Click send. You will get the information you want in json format else, there is a bug in your code or the server is not running

Any background context you want to provide?

The server code is written using flask mini-framework

What are the relevant pivotal tracker stories?

#160231895
